tests/sqlite3/test-sqlite3.cpp

```diff
@@ -470,10 +471,10 @@ TEST_CASE("SQLite type inference", "[sqlite][sequence]")
         row const& r1 = (*it);
         CHECK(r1.get<std::string>(0) == cvc);
         CHECK(r1.get<double>(1) == Approx(cdec));
-        CHECK(r1.get<long long>(2) == cll);
-        CHECK(r1.get<unsigned long long>(3) == cull);
-        CHECK(r1.get<long long>(4) == cll);
-        CHECK(r1.get<unsigned long long>(5) == cull);
+        CHECK(r1.get<int64_t>(2) == cll);
+        CHECK(r1.get<uint64_t>(3) == cull);
+        CHECK(r1.get<int64_t>(4) == cll);
+        CHECK(r1.get<uint64_t>(5) == cull);
```

Comment:
This is an example of a change I'd prefer to undo, it doesn't do anything useful but just adds lines of code to review and results in unnecessary code churn in Git history.

Reply:
A problem I'm having with this is that the `soci::row` API seems to enforce the type that is allowed for `T` in `row.get<T>(index)` as defined in the respective backend. Undoing changes like these results in failing tests because of `std::bad_cast` exceptions.

Reply:
Nevermind, I fixed it and reverted the changes here.